docs/default.nix

```diff
@@ -97,6 +105,9 @@ let
 in
 {
   inherit options-json;
+  inherit (pkgsDoc) nixos-render-docs;
+
+  gfm-alerts-to-admonitions = pkgsDoc.python3.pkgs.callPackage ./gfm-alerts-to-admonitions { };
```

Comment:
I wonder if `gfm-admonitions-to-nixpkgs` is a better name?

Admonitions is a generic term, but the tokens we're targeting are nixpkgs-sepecific.

Reply:
I feel like `nixpkgs` is too vague, it would have to be `nixos-render-docs` to be descriptive enough, but it's a bit long.

The plugin itself does not really rely on nixos-render-docs though, it creates `admonition_open` and `admoninition_close` tokens.
How to parse them is orthogonal to this plugin, so the name still makes sense for me